gradle.properties

```diff
@@ -7,14 +7,14 @@ pluginRepositoryUrl=https://github.com/bufbuild/intellij-buf
 pluginVersion=0.2.1
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild=223
+pluginSinceBuild=233
 pluginUntilBuild=999.*
 # IntelliJ Platform Properties -> https://github.com/JetBrains/gradle-intellij-plugin#intellij-platform-properties
 platformType=IU
-platformVersion=2022.3.3
+platformVersion=2023.3
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
-platformPlugins=idea.plugin.protoeditor, org.jetbrains.plugins.yaml
+platformPlugins=idea.plugin.protoeditor, io.kanro.idea.plugin.protobuf:1.7.50, org.jetbrains.plugins.yaml
```

Comment:
How will this impact users installing this plugin? Will they be asked to install one or the other?

Reply:
No, those deps used for compiling and coding.

Reply:
Users just need install one of protobuf plugins can be work